### ISSUE TITLE
why loss_l is nan?

### ISSUE THREAD

**Issue by LilacYue** [ruinmessi/RFBNet#72]
pytorch 0.4.0 python3.6
when i train 2-class detection,  loss_l will be nan when Totel iter is 11
log is like this:
Epoch:1 || epochiter: 5/489|| Totel iter 5 || L: 4.0048 C: 10.7443 ||Batch time: 1.1098 sec. ||LR: 0.00000918
Epoch:1 || epochiter: 6/489|| Totel iter 6 || L: 3.7286 C: 9.1246 ||Batch time: 1.1129 sec. ||LR: 0.00001081
Epoch:1 || epochiter: 7/489|| Totel iter 7 || L: 4.2147 C: 9.2150 ||Batch time: 1.1127 sec. ||LR: 0.00001245
Epoch:1 || epochiter: 8/489|| Totel iter 8 || L: 3.6471 C: 8.7142 ||Batch time: 1.2073 sec. ||LR: 0.00001408
Epoch:1 || epochiter: 9/489|| Totel iter 9 || L: 4.7703 C: 8.9132 ||Batch time: 1.1896 sec. ||LR: 0.00001572
Epoch:1 || epochiter: 10/489|| Totel iter 10 || L: 4.1172 C: 6.9407 ||Batch time: 1.2404 sec. ||LR: 0.00001736
Epoch:1 || epochiter: 11/489|| Totel iter 11 || L: nan C: 7.5253 ||Batch time: 1.2383 sec. ||LR: 0.00001899
Epoch:1 || epochiter: 12/489|| Totel iter 12 || L: nan C: 6.7064 ||Batch time: 1.2252 sec. ||LR: 0.00002063
Epoch:1 || epochiter: 13/489|| Totel iter 13 || L: nan C: 5.6455 ||Batch time: 1.2202 sec. ||LR: 0.00002226
Epoch:1 || epochiter: 14/489|| Totel iter 14 || L: nan C: 5.0595 ||Batch time: 1.2255 sec. ||LR: 0.00002390
Epoch:1 || epochiter: 15/489|| Totel iter 15 || L: nan C: 4.8662 ||Batch time: 1.2223 sec. ||LR: 0.00002553
Epoch:1 || epochiter: 16/489|| Totel iter 16 || L: nan C: 4.4298 ||Batch time: 1.2331 sec. ||LR: 0.00002717
Epoch:1 || epochiter: 17/489|| Totel iter 17 || L: nan C: 4.5185 ||Batch time: 1.2298 sec. ||LR: 0.00002880
Epoch:1 || epochiter: 18/489|| Totel iter 18 || L: nan C: 4.3470 ||Batch time: 1.2306 sec. ||LR: 0.00003044
Epoch:1 || epochiter: 19/489|| Totel iter 19 || L: nan C: 4.0623 ||Batch time: 1.2324 sec. ||LR: 0.00003208
Epoch:1 || epochiter: 20/489|| Totel iter 20 || L: nan C: 3.9098 ||Batch time: 1.2308 sec. ||LR: 0.00003371
Epoch:1 || epochiter: 21/489|| Totel iter 21 || L: nan C: 4.0042 ||Batch time: 1.2342 sec. ||LR: 0.00003535
Epoch:1 || epochiter: 22/489|| Totel iter 22 || L: nan C: 3.8657 ||Batch time: 1.2349 sec. ||LR: 0.00003698
Epoch:1 || epochiter: 23/489|| Totel iter 23 || L: nan C: 3.7135 ||Batch time: 1.2300 sec. ||LR: 0.00003862
Epoch:1 || epochiter: 24/489|| Totel iter 24 || L: 2.7975 C: 3.8324 ||Batch time: 1.2299 sec. ||LR: 0.00004025
Epoch:1 || epochiter: 25/489|| Totel iter 25 || L: nan C: 3.6185 ||Batch time: 1.2252 sec. ||LR: 0.00004189
Epoch:1 || epochiter: 26/489|| Totel iter 26 || L: nan C: 3.6493 ||Batch time: 1.2332 sec. ||LR: 0.00004353
Epoch:1 || epochiter: 27/489|| Totel iter 27 || L: nan C: 3.2827 ||Batch time: 1.2135 sec. ||LR: 0.00004516
Epoch:1 || epochiter: 28/489|| Totel iter 28 || L: nan C: 3.0109 ||Batch time: 1.2090 sec. ||LR: 0.00004680
Epoch:1 || epochiter: 29/489|| Totel iter 29 || L: nan C: 3.5017 ||Batch time: 1.2120 sec. ||LR: 0.00004843
Epoch:1 || epochiter: 30/489|| Totel iter 30 || L: nan C: 2.9969 ||Batch time: 1.2101 sec. ||LR: 0.00005007
Epoch:1 || epochiter: 31/489|| Totel iter 31 || L: nan C: 3.1932 ||Batch time: 1.2119 sec. ||LR: 0.00005170
Epoch:1 || epochiter: 32/489|| Totel iter 32 || L: nan C: 2.8617 ||Batch time: 1.2138 sec. ||LR: 0.00005334
Epoch:1 || epochiter: 33/489|| Totel iter 33 || L: nan C: 3.1685 ||Batch time: 1.2112 sec. ||LR: 0.00005497
Epoch:1 || epochiter: 34/489|| Totel iter 34 || L: nan C: 3.0457 ||Batch time: 1.2094 sec. ||LR: 0.00005661
Epoch:1 || epochiter: 35/489|| Totel iter 35 || L: nan C: 3.0445 ||Batch time: 1.2076 sec. ||LR: 0.00005825
Epoch:1 || epochiter: 36/489|| Totel iter 36 || L: nan C: 3.3814 ||Batch time: 1.2003 sec. ||LR: 0.00005988

what should I do

**Comment by ZHANGHeng19931123** [comment]
Same problem here, looking forward to solutions! Thanks

**Comment by pxjx** [comment]
Similar problem here.
When I train two class detection, my location loss in epoch 1 is about 3.9 which is slow. But in epoch 1600, the location loss drops slightly compare with the location loss in epoch 1.

**Comment by ys0823** [comment]
I have met the same problem with you. Because some of my train data have none objects .After I delete these samples,it 's OK.

**Comment by JimmyTang178** [comment]
I encountered the same problem, maybe in your annotations the xmin or ymin is larger than xmax or ymax. I just check my ground truth and make all the min<max.
